src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/FormattingUtilities.cs

```diff
@@ -260,4 +262,22 @@ static bool IsWhitespace(char c)
             return builder.DrainToImmutable();
         }
     }
+
+    /// <summary>
+    /// Sometimes the Html language server will send back an edit that contains a tilde, because the generated
+    /// document we send them has lots of tildes. In those cases, we need to do some extra work to compute the
+    /// minimal text edits
+    /// </summary>
+    // Internal for testing
```

Comment:
Is this comment still true?

Reply:
Whoops, hasn't been true for a while. Sneaky little thing hiding with the Xml comments!